babashka-http-client/test/martian/babashka/http_client_test.clj

```diff
@@ -181,3 +197,168 @@
       (require '[clojure.test] '[martian.babashka.http-client-test])
       (let [{:keys [error fail]} (clojure.test/run-tests 'martian.babashka.http-client-test)]
         (when (or (pos? error) (pos? fail)) (System/exit 1)))"))))
+
+(deftest multipart-request-test
+  (let [m (martian-http/bootstrap-openapi openapi-multipart-url)]
+
+    (testing "common types:"
+      (testing "String"
+        (is (= {:version :http-1.1
+                :method :post
+                :url "http://localhost:8888/upload"
+                :multipart [{:name "string" :content "Howdy!"}]
+                :headers {"Accept" "application/json"}
+                :as :text}
+               (martian/request-for m :upload-data {:string "Howdy!"})))
+        (is (match?
+              {:version :http1.1
+               :status 200
+               :headers {:content-type "application/json;charset=utf-8"}
+               :body {:content-type multipart+boundary?
```

Comment:
We seem to have different behaviour here depending on whether version is `http-1.1` or `http1.1` which look semantically the same to me - would you mind explaining what the difference is here?

I also can't see how an identical function call is resulting in the two different `:version` values?

Reply:
Still confused by the inconsistency between `http-1.1` and `http1.1`

Reply:
This is a `babashka-http-client`-specific thing (as you can see [here](https://github.com/oliyh/martian/blob/master/babashka-http-client/src/martian/babashka/http_client.clj#L28-L29)), so now I regret that I didn't remove it completely, but left strict equality here and there. Since this has nothing to do with the task at hand or the values ​​being tested, I will remove all `:version`s.